src/pages/Products.jsx

```diff
@@ -1,21 +1,46 @@
 import React from "react";
+import { useState, useEffect } from "react";
 import Product from "../components/products/Product";
 import ProductFilter from "../components/products/ProductFilter";
 import TopBar from "../components/products/TopBar";
```

Comment:
its increasing lines code I think you forgot to single line import from ```index.js```

Reply:
Thanks for the suggestion